packages/amplication-client/src/Purchase/PurchasePage.tsx

```diff
@@ -70,6 +70,7 @@ const PurchasePage = (props) => {
     });
 
   const handleContactUsClick = useCallback(() => {
+    history.push("?contact-us=true");
```

Comment:
Not part of this PR - but I think the event's name is not good enough - "contact us" is generic and already used in the top bar. 
Please change it to "PurchaseContactUsButtonClicked" (new event - do not change the name of the current event if it is used in other places)